src/darksend-relay.cpp

```diff
@@ -99,12 +99,12 @@ void CDarkSendRelay::Relay()
 
 void CDarkSendRelay::RelayThroughNode(int nRank)
 {
-    CMasternode* pmn = mnodeman.GetMasternodeByRank(nRank, nBlockHeight, MIN_PRIVATESEND_PEER_PROTO_VERSION);
+    masternode_info_t mnInfo;
 
-    if(pmn != NULL){
-        //printf("RelayThroughNode %s\n", pmn->addr.ToString().c_str());
+    if(!mnodeman.GetMasternodeByRank(nRank, nBlockHeight, MIN_PRIVATESEND_PEER_PROTO_VERSION, false, mnInfo)){
```

Comment:
If the negation desired here? mnInfo will be uninitialized in case of GetMasternodeByRank returning false.